kv/dist_sender.go

```diff
@@ -353,8 +351,7 @@ func (ds *DistSender) sendRPC(trace *tracer.Trace, rangeID proto.RangeID, replic
 	var addrs []net.Addr
 	replicaMap := map[string]*proto.Replica{}
 	for i := range replicas {
```

Comment:
if you turn this into an iterator instead of just a for loop, it will clean up the code a bit.
`for _, replica := range replicas {`